coordinator/gscoordinator/utils.py

```diff
@@ -1987,3 +1987,14 @@ def _check_task(endpoint):
         if time.time() - begin_time > INTERAVTIVE_INSTANCE_TIMEOUT_SECONDS:
             executor.shutdown(wait=False)
             raise TimeoutError(f"Gremlin check query failed: {error_message}")
+
+
+def check_java_installation():
+    if shutil.which("java") is None:
+        if os.environ.get("JAVA_HOME", None) is not None:
+            _java_executable = os.path.join(os.environ.get("JAVA_HOME"), "bin", "java")
+        if not os.path.isfile(_java_executable) or not os.access(
+            _java_executable, os.X_OK
+        ):
+            return False
+    return True
```

Comment:
We already have such utility: https://github.com/alibaba/GraphScope/blob/main/python/graphscope/framework/utils.py#L189-L197

Reply:
This method should be deleted.